python/pyrogue/__init__.py

```diff
@@ -66,11 +67,23 @@ def addLibraryPath(path):
 def waitCntrlC():
     """Helper Function To Wait For Cntrl-c"""
 
-    print("Running. Hit cntrl-c to exit.")
+    class monitorSignal(object):
+        def __init__(self):
+            self.runEnable = True
+
+        def receiveSignal(self,*args):
+            print("Got SIGTERM, exiting")
+            self.runEnable = False
+
+    mon = monitorSignal()
+    signal.signal(signal.SIGTERM, mon.receiveSignal)
+
+    print(f"Running. Hit cntrl-c or send SIGTERM to {os.getpid()} to exit.")
     try:
-        while True:
-            time.sleep(1)
+        while mon.runEnable:
+            time.sleep(0.5)
```

Comment:
why did the sleep change from 1 second to 500 ms?

Reply:
I wanted the response to be a little more immediate to avoid double kills.